Podstawy programowania/Ćwiczenia powtórzeniowe/39_Kho_Zad_powt_PodstProg2.py

```diff
@@ -0,0 +1,85 @@
+import math
+import cmath
+
+
+# 1
+def get_number(text):
+    num = input(text)
+    while True:
+        try:
+            num = float(num)
+            break
+        except:
+            num = input("Błędna wartość. Spróbuj ponownie:\n")
+    return num
+
+
+print("Mamy dane równanie ax + b = 0")
+a = get_number("Podaj a:\n")
+b = get_number("Podaj b:\n")
+if a == 0:
+    print("Brak rozwiązania.")
```

Comment:
Technicznie to nie brak rozwiązania tylko:
- brak rozwiązania, gdy b != 0
- nieskończenie wiele rozwiązań, gdy b == 0